metricbeat/module/kubernetes/state_deployment/state_deployment.go

```diff
@@ -32,7 +32,23 @@ var mapping = &p.MetricsMapping{
 		"kube_deployment_status_replicas_unavailable": p.Metric("replicas.unavailable"),
 		"kube_deployment_status_replicas_available":   p.Metric("replicas.available"),
 		"kube_deployment_spec_replicas":               p.Metric("replicas.desired"),
-		"kube_deployment_spec_paused":                 p.BooleanMetric("paused"),
+		/*
+			This is how deployment_status_condition field will be exported:
+
+			kube_deployment_status_condition{namespace="default",deployment="test-deployment",condition="Available",status="true"} 0
```

Comment:
I think it should be enough to add it in corresponding `plain` file - https://github.com/elastic/beats/blob/ac1d127d6868caff2f1bbbc670bbfff7f88930fc/metricbeat/module/kubernetes/_meta/test/ksm.v2.4.2.plain, instead of this comment

Reply:
It is already there see [line 222](https://github.com/elastic/beats/blob/ac1d127d6868caff2f1bbbc670bbfff7f88930fc/metricbeat/module/kubernetes/_meta/test/ksm.v2.4.2.plain#L222)

It is more alligned to this comment: https://github.com/elastic/beats/blob/ac1d127d6868caff2f1bbbc670bbfff7f88930fc/metricbeat/module/kubernetes/state_job/state_job.go#L41-L46

I found it helpful when first looked the code, because you can not understand how the p.LabelMatric works from first glance.

Let me know if you think is obsolete and I can remove it